q2galaxy/core/templaters/common.py

```diff
@@ -115,3 +115,10 @@ def make_formats_help(formats):
             help_ += f' - {format_.__name__}\n'
 
     return help_
+
+
+def make_xrefs():
+    xefs = XMLNode('xefs')
+    xefs.append(
+        XMLNode('xref', 'qiime2', type='bio.tools'))
+    return xefs
```

Comment:
```suggestion
    xrefs = XMLNode('xrefs')
    xrefs.append(
        XMLNode('xref', 'qiime2', type='bio.tools'))
    return xrefs
```
A tiny typo here, galaxy XML spec seems to expect `xref` (as does the order of `OrderTool`)